mesa/__init__.py

```diff
@@ -9,8 +9,12 @@
 from mesa.model import Model
 from mesa.agent import Agent
 
+import mesa.time as time
```

Comment:
Actually, here is another point that requires a consensus.

Should we do
```
import mesa
...
mesa.time.RandomActivation
# or
mesa.RandomActivation
```
?

Reply:
I chose `mesa.time.RandomActivation`, because I want it to be consistent that other groupings, such as `mesa.visualization.Slider`, require a namespace prefix. To the reader of the code, `mesa.visualization.Slider` gives better context of what it is, more than `mesa.Slider`.

Reply:
The trade-off here is that `mesa.Slider` is comparatively easier to type for quick prototyping in a Jupyter notebook.

And so, I think we can have the namespace hierarchy in this PR as the default, for the general use case, and still have #1244 with flat namespace (no `time`, `space`, `visualization` grouping; everything is flat) for the quick prototyping use case.